CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 2.3.0 TBD
+
+- Change: Outbound connections are now routed to intercepted agents.
+- Change: Inbound connection from intercepted agent now uses existing gRPC instead of new connection to manager port between 6000-7999
```

Comment:
This sentence reads super weird to me.  Perhaps:

> - Change: Inbound connections from an intercepted agent are now tunneled to the manager over the existing gRPC connection, instead of establishing a new connection to the manager for each inbound connection.